forch/forchestrator.py

```diff
@@ -74,7 +75,7 @@ def initialize(self):
         self._cpn_collector = CPNStateCollector()
 
         self._faucet_config_file = os.path.join(
-            os.getenv('FAUCET_CONFIG_DIR'), _FAUCET_CONFIG_DEFAULT)
+            os.getenv('FAUCET_CONFIG_DIR'), _DYNAMIC_CONFIG_DEFAULT)
```

Comment:
this doesn't seem right -- why is this DYNAMIC config?  How is this entry sync'd with other places the dynamic_config is specified?  Maybe rename varaible to self._structuctural_config_file or self._behavorial_config_file to disambiguate cleanly?  I would make sure that the value is calculated in exactly one place, and cached and used elsewhere (i.e. _DYANMIC_CONFIG_DEFAULT should only be used in one place)

Reply:
So self._faucet_config_file is the file that is read by faucet. It will be used in _restore_states() and get_sys_config(). So I think here it should be dynamic config file instead of structural config file?

Reply:
Changed _DYNAMIC_CONFIG_DEFAULT back to _FAUCET_CONFIG_DEFAULT, and set self._faucet_config_file in _initialize_faucetizer(). Idea is that if forch is configured to run faucetizer, then it will use the dynamic file as self._faucet_config_file. Otherwise, the self._faucet_config_file will be initialized as before.

Reply:
Renamed self._faucet_config_file to self._behavior_config_file